src/LaunchDarkly/ApcuLDDFeatureRequester.php

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
APC and APCU are different extensions. please correct me if i'm wrong; APC is only available up to php54. beyond that, the docs recommend usage of the built-in op-code cache. so a compatible APCU extension was created for >=php56. my change will allow integrations to choose APC or APCU based on which extension they have installed and should cover a wider range of PHP versions. i felt i needed to do this as a part of this merge request because i'm trying to get the integration tests to run on CircleCI for the default php56 version, which does not support APC (i think). the integration tests that run in the Vagrant VM switch the PHP version to php54, which is fine, but i think we should be able to run integration tests on other php versions.

i'd be happy to update the docs with use cases and examples. didn't know if this was a welcome proposal. also, the integration tests are failing the build, for a couple of reasons. one is that monolog needs to be installed (addressed here https://github.com/launchdarkly/php-client/pull/55) and the other (i think) is that the feature-generator function in the integration tests needs to be updated. i took a crack at it in another pull request, but i felt it was getting bloated. thoughts on how to proceed?